src/AccessibilityInsights/Modes/ConfigurationModeControl.xaml.cs

```diff
@@ -170,11 +179,13 @@ private async Task<bool> HandleReleaseSwitch()
                 ex.ReportException();
 
                 Dispatcher.Invoke(() =>
-                    MessageDialog.Show(string.Format(CultureInfo.InvariantCulture, Properties.Resources.ConfigurationModeControl_VersionSwitcherException))
-                );
-            }
+                {
+                    MainWin.Topmost = oldTopMost;
+                    MessageDialog.Show(string.Format(CultureInfo.InvariantCulture, Properties.Resources.ConfigurationModeControl_VersionSwitcherException));
```

Comment:
I'm assuming we call String.Format here because there's no other way to indicate invariant culture? This looks pretty strange at first glance.

Reply:
Actually, we were calling string.Format because we used to be replacing format items in the string. We stopped doing that, but never removed the unneccessary string.format call. Removing it now!